spark3.0/project/build.properties

```diff
@@ -0,0 +1 @@
+sbt.version = 1.3.8
```

Comment:
Use the latest version of sbt 1.4.7

Reply:
Upgraded in commit https://github.com/splicemachine/splice-machine-spark-connector/pull/31/commits/d39bb4f6fdad8aa5baa8f50d05ab3ff7b248d865